DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/AutocadContext.cs

```diff
@@ -0,0 +1,3 @@
+namespace Speckle.Connectors.Autocad.HostApp;
+
+public class AutocadContext { }
```

Comment:
We should probably take this through if this is intended to be part of that ConversionContext pattern, or it may be for ACad it isn't needed in the way the RevitContext is needed?